api/v1alpha1/usersignup_types.go

```diff
@@ -174,7 +174,6 @@ type UserSignupSpec struct {
 	States []UserSignupState `json:"states,omitempty"`
 
 	// IdentityClaims contains as-is claim values extracted from the user's access token
-	// +optional
 	IdentityClaims IdentityClaimsEmbedded `json:"identityClaims,omitempty"`
```

Comment:
I guess we don't need `omitempty` for required fileds?
```suggestion
	IdentityClaims IdentityClaimsEmbedded `json:"identityClaims"`
```

Reply:
Fixed.